server.go

```diff
@@ -1325,12 +1326,41 @@ func newServer(cfg *Config, listenAddrs []net.Addr,
 		cfg.HealthChecks.DiskCheck.Attempts,
 	)
 
+	tlsHealthCheck := healthcheck.NewObservation(
```

Comment:
Also need to add this new cfg to our validation: https://github.com/lightningnetwork/lnd/blob/master/lncfg/healthcheck.go#L32